src/main/webapp/js/components/dev/ExperimentsTable/ExperimentsTable.js

```diff
@@ -0,0 +1,524 @@
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.RUNNING)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("RUNNING");
+                            tdStatus.attr("title", "RUNNING");
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.DESIGN)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("DESIGN");
+                            tdStatus.attr("title", "DESIGN");
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.QUEUED)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("QUEUED");
+                            tdStatus.attr("title", "QUEUED");
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.CANCELED)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("CANCELED");
+                            tdStatus.attr("title", "CANCELED");
+                        }
+                    }
+                }
+            });
+        },
+        
+		populate : function(){
+			var experiments = window.Project.getExperiments();
+			var rows = [];
+			var index = 0;
+			for ( var key in experiments)
+			{
+				var experiment = experiments[key];
+				rows[index] = experiment;
+				index++;
+			}
+			this.setState({ experiments: rows });
+		},
+
+		getInitialState: function() {
+			var tabledata = [];
+
+			return {experiments: tabledata, counter : 1};
+		},
+		onClick : function(rowID, e){
+			var targetRowId = "."+rowID;
+			if( $(targetRowId).is(':visible')){
+				$(targetRowId).hide();
+			}else{
+				$(targetRowId).show();
+			}
+		},
+		render: function() {
+			var style  = {
+					width:"30%"
+			};
+			var rows = [];
+			this.state.experiments.forEach(function(experiment) {
+				if(experiment!=null){
+					var expandableRowId = "collapsable-" +experiment.getId();
+					rows.push(<ExperimentRow experiment={experiment} rowNumber={this.state.counter} key={experiment.name} fnClick={this.onClick.bind(this,expandableRowId)}/>);
+					rows.push(<ExperimentExpandableRow experiment={experiment} rowNumber={this.state.counter} key={expandableRowId}/>);
+					this.state.counter++;
+				}
+			}.bind(this));
+
+					return (
+							<div className="col-lg-6 experimentsResults panel-body experimentsResultsOutput" id="experimentsOutput">
+								<table id="experimentsTable" className="table table-condensed experimentsTable" style={{borderCollapse:"collapse"}}>
+									<thead className="experimentsTableColumn">
+									<tr><th style={{width:"10%",textAlign:"center"}}>Status</th>
```

Comment:
Curious about this extra style information -- does it not work to have this in the less file?